api/config/db.js

```diff
@@ -0,0 +1,65 @@
+import { Sequelize } from "sequelize";
+import dotenv from "dotenv";
+
+// Load environment variables from .env file
+dotenv.config();
+
+// Validate required environment variables
+const requiredEnvVars = [
+  "DB_USER",
+  "DB_PASSWORD",
+  "DB_HOST",
+  "DB_PORT",
+  "DB_DATABASE_NAME",
+  "DB_CLIENT",
+];
+
+requiredEnvVars.forEach((envVar) => {
+  if (!process.env[envVar]) {
+    console.error(`Missing required environment variable: ${envVar}`);
+    process.exit(1); // Exit the process if any required variable is missing
+  }
+});
+
+// Debugging: Check if environment variables are loaded correctly
+console.log("DB_USER:", process.env.DB_USER);
```

Comment:
This is fine for local development, but logging database credentials is a security risk. 

I would suggest removing it, but if you guys want to keep it for easier development, you should check the `process.env.NODE_ENV` and only log it out for local development :D!

Reply:
true, i willl remove it, i logged it out because there waas a problem with the credentials in the deployment at railway